modules.cmake

```diff
@@ -128,22 +128,28 @@ endfunction()
 #
 # Usage:
 #   add_module_library(<name> [sources...] FALLBACK [sources...] [IF enabled])
-function(add_module_library name)
-  cmake_parse_arguments(AML "" "IF" "FALLBACK" ${ARGN})
-  set(sources ${AML_UNPARSED_ARGUMENTS})
+function(add_module_library)
```

Comment:
Changes to passing arguments are unrelated and breaking. Please revert.

Reply:
Resolved see: https://github.com/Islam0mar/modules/actions. It is fixed now.

I changed that to be easier to handle arguments, If you don't like that, I would revert.

Reply:
Please do.